OpenRA.Mods.Common/Widgets/Logic/Lobby/LobbyUtils.cs

```diff
@@ -33,6 +33,9 @@ public static class LobbyUtils
 		[FluentReference]
 		const string Bots = "options-lobby-slot.bots";
 
+		[FluentReference]
+		const string BotName = "label-bot-name";
```

Comment:
I still somewhat object to this variable name: this is the name of a player/client that is a bot, rather than the bot name itself.

Reply:
there's something wrong on my end, I didn't push all the changes again